File: api/src/main/java/equipmentManagementSystem/input/PUser.java
package equipmentManagementSystem.input;

/**
 * 对应user 用于修改手机号
 */
public class PUser {

    /**
     * 新手机号
     */
    private String newPhoneNumber;

    /**
     * 密码
     */

    private String password;


    public String getPassword() {
        return password;
    }

    public String getNewPhoneNumber() {
        return newPhoneNumber;
    }

    public void setNewPhoneNumber(String newPhoneNumber) {
        this.newPhoneNumber = newPhoneNumber;
    }
}

File: web/src/app/base/puser.ts
/**
 * 对应user（用户修改手机号）
 */
export class PUser {
  /** 新手机号 */
  newPhoneNumber: string;

  /** 新手机号验证码 */
  code: string;

  /** 原手机号验证码 */
  verificationCode: string;
}
